docker-compose.yml

```diff
@@ -103,6 +111,17 @@ services:
       args:
         GIT_HASH: null
 
+  omgwtfssl:
+    image: paulczar/omgwtfssl
+    restart: "no"
+    volumes:
+      - certs:/certs
+    environment:
```

Comment:
These settings seem to eliminate the idea that this docker-compose.yml can be used in production, no?